### DISCUSSION TITLE
Figuring out next steps

### DISCUSSION THREAD

**Discussion by keithschacht** [AllYourBot/hostedgpt#312] [General]
Hi @florianfelsing, I realized I don't have any other way to send you a note besides this so starting a thread just so I can tag you. :) I've been taking a step back with this project to think about where I want to go with it next. I have a few different ideas I'm considering and I'd really love to hear your thoughts. Want to connect on a zoom call sometime? If you see this, let's move to email to make it easier. Send me a note at krschacht at gmail.com. Thanks!

**Comment by florianfelsing** [comment]
Hey @krschacht, just emailed you. Sorry for getting back to you so late!
